static/src/stylesheets/module/commercial/_paidfor.scss

```diff
@@ -1,6 +1,12 @@
 /* ==========================================================================
    Paid for component
    ========================================================================== */
+
+.fc-container .fc-container__inner--paidfor {
```

Comment:
Specificity issues. I do not want to open a can of worms by trying to refactor the original module.